.chloggen/kubeletstats_fix_defaults.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: kubeletstatsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable .cpu.usage metrics by default
```

```suggestion
note: Enable CPU usage metrics by default
```
If we list all of metrics below, we can make this a more general user-friendly message.